extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -49,6 +50,11 @@ const SUPPORTED_ELEMENTS_ = {
   'amp-img': true,
   'amp-anim': true,
 };
+/** @private @const {!Object<string, boolean>} */
+const SUPPORTED_NATIVE_ELEMENTS = {
```

Comment:
Maybe consider a `Set` or `Map` for this; might be uglier though

Reply:
A `Set` makes sense, can add to this PR.